internal/command/default.provisioners.yaml

```diff
@@ -216,6 +216,96 @@
     - username
     - password
 
+
+- uri: template://default-provisioners/postgres
+  # By default, match all redis types regardless of class and id. If you want to override this, create another
+  # provisioner definition with a higher priority.
+  type: postgres-instance
+  description: Provisions a dedicated database on a shared postgres instance
+  # Init template has the random service name and password if needed later
+  init: |
+    randomServiceName: pg-{{ randAlphaNum 6 }}
+    randomDatabase: db-{{ randAlpha 8 }}
+    randomUsername: user-{{ randAlpha 8 }}
+    randomPassword: {{ randAlphaNum 16 | quote }}
+    sk: default-provisioners-postgres-instance
+    publishPort: {{ dig "annotations" "compose.score.dev/publish-port" "0" .Metadata | quote }}
+  # The state for each database resource is a unique db name and credentials
+  state: |
+    database: {{ dig "database" .Init.randomDatabase .State | quote }}
```

Comment:
In this usecase, wouldn't you want to be providing an admin account with access to the default database rather than a random one?